pkg/policy/condition/key.go

```diff
@@ -106,6 +118,13 @@ var CommonKeys = []Key{
 	AWSSecureTransport,
 	AWSCurrentTime,
 	AWSEpochTime,
+	AWSPrincipalType,
+	AWSUserID,
+	AWSUsername,
+}
+
+var varStr = func(k Key) string {
```

Comment:
this function need not be a separate.  `VarName()` is the actual function.

Reply:
done